### PR TITLE
Add Roles tasks and vars SLES

### PR DESCRIPTION
**Pull request by dh-roland**
For issue: #157 
Add Role Tasks and vars for SUSE

### CODE REVIEW

**Comment by dh-roland** [comment]
Recheck

**Comment by bobapple** [comment]
@cla-bot check

**Review by mkayontour — CHANGES_REQUESTED** [review]
Thanks for the PR, eveything looks great so far. Two changes should be made, firstly the cosmetics in the install.yml. And secondly please add a changelog fragment. 

Just create a yaml file in the `changelog/fragments` folder. Naming scheme should be something like `feature_things_change_at_installation.yml` and the content is similar to the following file: https://github.com/Icinga/ansible-collection-icinga/blob/main/changelogs/fragments/feature_monitoring_plugins.yml

Other than that, I'm happy to merge your PR

File: roles/icinga2/tasks/install_on_Suse.yml
[the remark this answers is not in the document]
Comment (mkayontour) [reply roles/icinga2/tasks/install_on_Suse.yml:1]: Additional can you please use the Ansible FQCN notation for the zypper modules. https://docs.ansible.com/ansible/latest/collections/community/general/zypper_module.html

**Review by mkayontour — APPROVED** [review]

**Comment by mkayontour** [comment]
I merged the PR, thank you very much for your contribution!

**Pull request merged**